src/scanner.c

```diff
@@ -0,0 +1,48 @@
+#include <tree_sitter/parser.h>
```

Comment:
Local include please, don't want another infinite loop bug using mismatched system-packaged header files ;)

```suggestion
#include "tree_sitter/parser.h"
```